js-watcher/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/julienschmidt/httprouter"
```

Comment:
Gin uses a custom version of this, see [here](https://gin-gonic.com/docs/benchmarks/). Also the library itself [says](https://github.com/julienschmidt/httprouter#web-frameworks-based-on-httprouter) that a bunch of other libraries that provide fancier functionality use it as a dependency. `julienschmidt/httprouter` has zero dependencies.

We do not need super-fancy route handling so I thought we can simply get away with using smaller requirements.

Reply:
I did not want to use gorillamux because they do not have a maintainer and according to the [readme](https://github.com/gorilla/mux#gorillamux) on their repo are still looking for one.